### DISCUSSION TITLE
My css file doesnt show up when i open up my github site

### DISCUSSION THREAD

**Discussion by Swagger111** [community/community#124181] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have made this unblocked games site on replit but i moved it here so it can run 24/7 but when i open my site my css file never loads for some reason i am new to coding and dont understand why. Please help me il put the link for the repostoriy below

https://github.com/Swagger111/Swaggerr111.github.io

**Comment by CopperEagle** [comment]
Hey there @Swagger111 

What stood out to me is Line 8. You should make the path relative. It is absolute right now. So, make "`/main.css`" to "`main.css`". Does this solve your problem?

**Reply by Swagger111** [reply]
Thanks man it did help me really apperciate it

**Reply by CopperEagle** [reply, marked as answer]
Great to hear :smile: Please consider marking my comment as the answer. Thanks!
